cmd/start-amazon-cloudwatch-agent/start-amazon-cloudwatch-agent.go

```diff
@@ -92,8 +93,25 @@ func main() {
 		log.Fatalf("E! Cannot translate JSON config into TOML, ERROR is %v \n", err)
 	}
 	log.Printf("I! Config has been translated into TOML %s \n", tomlConfigPath)
+	echoTOML(tomlConfigPath)
```

Comment:
My two cents told me we should not log Toml File. What would be the result of printing the Toml File?
1. For customer, JSON Configuration would be enough for them to consider before moving to Toml File 
2. For us, TOML Configuration only helps us debug which is not always the time and we could do it by ourself instead of logging for our customer.